reflex/utils/types.py

```diff
@@ -194,6 +195,13 @@ def get_attribute_access_type(cls: GenericType, name: str) -> GenericType | None
                 return List[class_]
             else:
                 return class_
+        if isinstance(attr, AssociationProxyInstance):
+            return list[
```

Comment:
do we need cap `List` here or will this work fine on 3.8?

Not sure how to test it, but I assume you have the environment to confirm.

Reply:
I can not test with 3.8 easily. But I will test if it works with `List` as well. Sorry, I always forget about 3.8

Reply:
I changed it to cap `List` and it still works fine with recent python versions